admin-dev/themes/default/js/calendar.js

```diff
@@ -320,7 +320,7 @@ $(document).ready(() => {
     }
   }).data('daterangepicker');
 
-  const datepickerEnd = $('.datepicker2').daterangepicker({
+  datepickerEnd = $('.datepicker2').daterangepicker({
```

Comment:
```suggestion
  let datepickerEnd = $('.datepicker2').daterangepicker({
```

Reply:
@Progi1984 as datepickerEnd is already declared below, you don't need to declare it again as it's inside the same scope